TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+### Add track data for the preview site btn  #6623
```

Comment:
I feel testing instruction is unnecessary for this PR since we're not adding any new feature.

Reply:
@ilyasfoo that makes sense. Removed in https://github.com/woocommerce/woocommerce-admin/pull/6623/commits/b55c746320a411c07eb456f7da3788aeeacb9572